check.sh

```diff
@@ -0,0 +1,13 @@
+echo "Cloning opentelemetry-proto folder"
+svn checkout https://github.com/open-telemetry/opentelemetry-proto/trunk/opentelemetry
+mkdir proto
+protoc --proto_path=./ --php_out=proto opentelemetry/proto/collector/logs/v1/logs_service.proto
+protoc --proto_path=./ --php_out=proto opentelemetry/proto/collector/metrics/v1/metrics_service.proto
```

Comment:
Is there a reason you defined each of these differently rather than using wildcards?